### PR TITLE
Update extraction output

### PR DESCRIPTION
**Pull request by CedricThomas**

### CODE REVIEW

File: src/constructor/job.ts [hunk 1]
```diff
@@ -15,9 +15,9 @@ import {
     ExtractionBatchCSV,
     SourceDocument,
     RenderingJob,
-    PaginatedRenderingJobs,
+    PaginatedRenderingJobs, ExtractionJobOuput,
```
Comment (ArthurKnoep) [inline src/constructor/job.ts:18]: should be on new line

**Review by ArthurKnoep — CHANGES_REQUESTED** [review]
nope

**Review by ArthurKnoep — APPROVED** [review]
LGTM

**Comment by Hirevo** [comment]
:tada: The PR has just been merged ! Hooray !! :tada:<br/><br/>*This message is **definitely** not automated, please don't ban me.*

**Pull request merged**